test/test.js

```diff
@@ -637,30 +576,32 @@ describe('proactive token refresh', () => {
 describe('client options', () => {
   beforeEach(() => {
     fetchMock.restore();
-    fetchMock.post('https://stitch2.mongodb.com/api/client/v1.0/app/testapp/auth/local/userpass', {userId: hexStr});
-    fetchMock.post('https://stitch2.mongodb.com/api/client/v1.0/app/testapp/pipeline', (url, opts) => {
-      return {result: [{x: {'$oid': hexStr}}]};
+    fetchMock.post('https://stitch2.mongodb.com/api/client/v2.0/app/testapp/auth/providers/local-userpass/login', {user_id: hexStr});
+    fetchMock.post('https://stitch2.mongodb.com/api/client/v2.0/app/testapp/functions/call', (url, opts) => {
+      return {x: {'$oid': hexStr}};
+    });
+    fetchMock.delete(SESSION_URL, {});
+    fetchMock.post(FUNCTION_CALL_URL, () => {
+      return JSON.stringify({x: {'$oid': hexStr}});
     });
-    fetchMock.delete(BASEAUTH_URL, {});
-    fetchMock.post(PIPELINE_URL, {result: [{x: {'$oid': hexStr}}]});
   });
 
   it('allows overriding the base url', () => {
     let testClient = new StitchClient('testapp', {baseUrl: 'https://stitch2.mongodb.com'});
     expect.assertions(1);
     return testClient.login('user', 'password')
       .then(() => {
-        return testClient.executePipeline([{action: 'literal', args: {items: [{x: {'$oid': hexStr}}]}}]);
+        return testClient.executeFunction('testfunc', {items: [{x: {'oid': hexStr}}]}, 'hello');
       })
       .then((response) => {
-        expect(response.result[0].x).toEqual(new ejson.bson.ObjectID(hexStr));
+        expect(response.x).toEqual(new ejson.bson.ObjectID(hexStr));
       });
   });
 
   it('returns a rejected promise if trying to execute a pipeline without auth', (done) => {
     let testClient = new StitchClient('testapp');
     testClient.logout()
-      .then(() => testClient.executePipeline([{action: 'literal', args: {items: [{x: {'$oid': hexStr}}]}}]))
+      .then(() => testClient.executeFunction('testfunc', {items: [{x: {'oid': hexStr}}]}, 'hello'))
```

Comment:
why `oid` instead of `$oid`?

Reply:
Took this from previous usages of executeFunction. Will change.